src/NuGet.Jobs.Auxiliary2AzureSearch/Job.cs

```diff
@@ -12,6 +13,7 @@ namespace NuGet.Jobs
     public class Job : AzureSearchJob<Auxiliary2AzureSearchCommand>
     {
         private const string ConfigurationSectionName = "Auxiliary2AzureSearch";
+        private const string FeatureFlagConfigurationSectionName = "FeatureFlags";
```

Comment:
Thoughts on putting this in `AzureSearchJob<T>` instead?

Reply:
Done